Libraries/LayoutAnimation/LayoutAnimation.js

```diff
@@ -74,6 +74,12 @@ function configureNext(config: Config, onAnimationDidEnd?: Function) {
   UIManager.configureNextLayoutAnimation(
     config, onAnimationDidEnd || function() {}, function() { /* unused */ }
   );
+  // Clear the layout animation configuration after the current frame.
+  requestAnimationFrame(() => {
+    UIManager.configureNextLayoutAnimation(
+      null, function() { /* unused */ }, function() { /* unused */ }
+    );
+  });
 }
 
 function create(duration: number, type, creationProp): Config {
```

Comment:
Should we support setting a different prop for create and delete or this seems fine?